man/systemd.network.xml

```diff
@@ -2961,6 +2961,23 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             <varname>BitRate=</varname> is unspecified.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>TimeQuanta=</varname></term>
+          <term><varname>PropagationSegment=</varname></term>
+          <term><varname>PhaseBufferSegment1=</varname></term>
+          <term><varname>PhaseBufferSegment2=</varname></term>
+          <term><varname>SyncJumpWidth=</varname></term>
+          <listitem>
+            <para>Specifies the time quanta, propagation segment, phase buffer segment 1 and 2, and the
+            synchronization jump width, which allow to define the CAN bit-timing in a hardware
+            independent format as proposed by the Bosch CAN 2.0 Specification.
+            <varname>TimeQuanta=</varname> takes a timespan. <varname>PropagationSegment=</varname>,
+            <varname>PhaseBufferSegment1=</varname>, <varname>PhaseBufferSegment2=</varname>, and
+            <varname>SyncJumpWidth=</varname> takes an unsigned integer in the range 0…4294967295 in
+            the units of the time quanta. These settings except for <varname>SyncJumpWidth=</varname>
```

Comment:
so these these segments are factors to TimeQuanta? maybe say so, i.e. "in the units of the time quanta" → "in multiples of the time quanta"

Reply:
No, these settings take the number of time quantum.